error: this task is defined multiple times
 --> ui/task-interrupt.rs:9:14
  |
9 |     async fn foo(_: foo::Context) {}
  |              ^^^
